Stripe/STPAPIClient+Private.h

```diff
@@ -41,7 +41,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 - (NSURLSessionDataTask *)retrieveSourceWithId:(NSString *)identifier
                                   clientSecret:(NSString *)secret
-                            responseCompletion:(STPAPIResponseBlock)completion;
+                            responseCompletion:(void (^)(STPSource * _Nullable, NSHTTPURLResponse * _Nullable, NSError * _Nullable))completion;
```

Comment:
If you're sure about this breaking change, you should at least define this new type so that it can be reused by name rather than requiring a full signature match.

Consider: https://github.com/stripe/stripe-ios/blob/7152797d123b20fd3bee8e29ab84b9bf3e9227a2/Stripe/PublicHeaders/STPBlocks.h